sass/includes/font.scss

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: "et-book";
+  src: url("public/fonts/et-book-roman-line-figures.ttf") format("truetype");
```

Comment:
I added the "public/fonts/et-book-roman-line-figures.ttf" file, but I wasn't able to fully implement it as I'm not familiar with Rust. If you would like to implement the functionality yourself or have any other suggestions, please let me know. Alternatively, you can link to the font hosted on the font's repository:
```css
src: url("https://edwardtufte.github.io/et-book/et-book/et-book-roman-line-figures/et-book-roman-line-figures.ttf") format("truetype");
```